File: app/src/main/java/com/apps/wow/droider/Utils/AppContext.java
package com.apps.wow.droider.Utils;

import android.content.Context;

/**
 * Created by Jackson on 14/05/2017.
 */

public class AppContext {

    static Context mContext;

    public AppContext(Context mContext) {
        this.mContext = mContext;
    }

    public static Context getContext() {
        return mContext;
    }
}

File: app/src/main/java/com/apps/wow/droider/Utils/AppContext.kt
package com.apps.wow.droider.Utils

import android.annotation.SuppressLint
import android.content.Context

/**
 * Created by Jackson on 14/05/2017.
 */

class AppContext(mContext: Context) {

    init {
        Companion.context = mContext
    }

    companion object {
        @SuppressLint("StaticFieldLeak")
        lateinit var context: Context internal set
    }
}
